symplyphysics/laws/dynamics/forced_non_resonant_oscillations.py

```diff
@@ -53,7 +55,26 @@
     / (natural_angular_frequency**2 - driving_angular_frequency**2)
 )
 
-# TODO: prove this is a solution of the driven oscillations equation
+# Derive law from forced oscillations equation
+
+_law = forced_eqn.law
+
+_eqn = _law.rhs - _law.lhs
+_eqn = _eqn.subs({
+    forced_eqn.oscillator_mass: oscillator_mass,
+    forced_eqn.natural_angular_frequency: natural_angular_frequency,
+    forced_eqn.driving_force_amplitude: driving_force_amplitude,
+    forced_eqn.driving_angular_frequency: driving_angular_frequency,
+    forced_eqn.driving_phase_lag: driving_phase_lag,
+    forced_eqn.time: time,
+})
+
+_dsolved = dsolve(_eqn, forced_eqn.displacement(time)).rhs
+
+# We're not interested in the general solution of the differential equation
+_dsolved = _dsolved.subs({"C1": 0, "C2": 0})
```

Comment:
Explain, what parts of solution we are throwing out.

Reply:
Okay